src/linklib.h

```diff
@@ -227,12 +227,12 @@ offsets[0] = 0;
 oldtypes[o] = MPI_LONG;
 blockcounts[0]= 2;
 
-MPI_Type_extent(MPI_LONG, &extent);
+MPI_Type_get_extent(MPI_LONG, &extent);
```

Comment:
This is missing the &lb parameter.

Reply:
Actually, that whole section is commented out, so it would compile regardless.

Reply:
I can add it there too just in case. Let me know.